src/gleam/erlang/process.gleam

```diff
@@ -71,6 +72,55 @@ pub fn new_subject() -> Subject(message) {
   Subject(owner: self(), tag: erlang.make_reference())
 }
 
+/// Checks to see whether a `Dynamic` value is a Subject, and return the Subject if
+/// it is.
+///
+/// ## Examples
+///
+/// ```gleam
+/// import gleam/dynamic
+///
+/// subject_from_dynamic(dynamic.from(new_subject()))
+/// // -> Ok(Subject))
+/// ```
+///
+/// ```gleam
+/// import gleam/dynamic
+///
+/// subject_from_dynamic(dynamic.from(123))
+/// // -> Error([
+/// //      DecodeError(expected: "Tuple of 3 elements", found: "Int", path: []),
+/// //    ])
+/// ```
+pub fn subject_from_dynamic(
```

Comment:
This function is unsound as it cannot verify that the message type is correct. It cannot be added to this library I'm afraid.

Reply:
Sure, it sounds right. I removed it and left only reference_from_dynamic.